src/App.js

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react';
 import { Route, Routes, useLocation } from 'react-router-dom';
-import SideBar from './components/sidebar/sidebar';
+import SideBar from './components/sidebar/sidebar.js';
```

Comment:
To get rid of the linter error:
- [ ] I found the issues with this, your import  `sidebar` is found in `./components/Sidebar/sidebar`and not `./components/sidebar/sidebar`.
- [ ] Also, you can ignore the extra `.js` extension. JavaScript should take care of that